ios/src/TGMapViewController.h

```diff
@@ -12,6 +12,26 @@
 #import <UIKit/UIKit.h>
```

Comment:
I also plan to "squash and merge" this PR, so it should say "hallahan with tallytalwar" when merged.